### PR TITLE
Change dates for Autumn 2020-21

### PR DESCRIPTION
**Pull request by Ayushk4**
I am setting Mid Sem to 19 Sept (Saturday) to 20 Sept(Sunday) just to not break the existing code bcoz of this online sem. Since there are no classes on those days, I expect this to work.
Sem start date = 1st Sept
End = 29th Nov (?)

### CODE REVIEW

**Comment by Ayushk4** [comment]
@Shankusu993 @thealphadollar Review pls.

File: dates.py [hunk 1]
```diff
@@ -6,10 +6,10 @@
 import sys
 
 # SEM_BEGIN=datetime.datetime.now(pytz.timezone('Asia/Kolkata'))
-SEM_BEGIN = build_event.generateIndiaTime(2020, 1, 2, 0, 0)
-MID_TERM_BEGIN = build_event.generateIndiaTime(2020, 2, 17, 0, 0)
-MID_TERM_END = build_event.generateIndiaTime(2020, 2, 25, 23, 59)
-END_TERM_BEGIN = build_event.generateIndiaTime(2020, 4, 20, 0, 0)
+SEM_BEGIN = build_event.generateIndiaTime(2020, 9, 1, 0, 0)
+MID_TERM_BEGIN = build_event.generateIndiaTime(2020, 9, 19, 0, 0)
```
Comment (thealphadollar) [inline dates.py:10]:
```suggestion
MID_TERM_BEGIN = build_event.generateIndiaTime(2020, 10, 23, 0, 0)
```

File: dates.py [hunk 2]
```diff
@@ -6,10 +6,10 @@
...
+MID_TERM_END = build_event.generateIndiaTime(2020, 9, 20, 23, 59)
```
Comment (thealphadollar) [inline dates.py:11]:
```suggestion
MID_TERM_END = build_event.generateIndiaTime(2020, 10, 27, 23, 59)
```

File: dates.py [hunk 3]
```diff
@@ -6,10 +6,10 @@
...
+END_TERM_BEGIN = build_event.generateIndiaTime(2020, 11, 29, 0, 0)
```
Comment (thealphadollar) [inline dates.py:12]:
```suggestion
END_TERM_BEGIN = build_event.generateIndiaTime(2020, 11, 26, 0, 0)
```

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Hey Ayush, thanks a lot for the PR. I had it on my schedule for today and I'm elated you were faster and more concerned :sparkling_heart: 

I've suggested some changes to the dates as per the [official calendar](http://www.iitkgp.ac.in/files/acad_cal2021.pdf). My apologies for saying 29 November in the chat when the winter break starts from 26th November. Also, I've changed the mid-sem dates to match with the autumn break dates.

Do you think you can find some time to reflect the same changes in [gyft-server](https://github.com/metakgp/gyft-serve/)?

File: dates.py (hunk 1 above)
Comment (Ayushk4) [reply dates.py:10]: What's wrong with the MID_TERM_BEGIN Date?

File: dates.py (hunk 2 above)
Comment (Ayushk4) [reply dates.py:11]: Ok. So, setting DP = Mid sem is nice.

**Review by thealphadollar — APPROVED** [review]
Thanks @Ayushk4

**Pull request merged**